### ISSUE TITLE
Git Workflow

### ISSUE THREAD

**Issue by fab9** [CrossTrack/crosstrackapp#10]
@LoganArnett  @Nhare  As I mentioned earlier, I'm not clear as to how we're using git. Below is the workflow I'm following atm and questions for both of you. :)
## My Workflow

_(Using git flow)_

**Start feature**
In my fork, branch off of `development`.

```
git flow feature start mobile-dashboard-view
```

**Finish feature**

```
git flow feature finish mobile-dashboard-view
```

 Feature is now merged into `development`. Delete local and remote (in my fork) feature branches.

**Push to my remote development branch**

```
 git push origin development
```

Then manually open PR from my remote `development` to main repo's `development` branch.
## Questions
1. Our latest code lives in main repo's `development`(MRD) branch, right? Nvm, it's not there. ??
2. Before starting a feature, are you pulling from MRD to get the latest code?
3. When finishing a feature, where are you pushing to?
4. Do you ever push feature branches to main repo or only to your local forks?
